<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>League_Manager_Main</class>
 <widget class="QMainWindow" name="League_Manager_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>League Manager</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="main_title_label">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Curling League Manager 2024</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="league_list"/>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="new_league_button">
          <property name="text">
           <string>New League</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="button_spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="edit_league_button">
          <property name="text">
           <string>Edit Leauge</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_league_button">
          <property name="text">
           <string>Remove League</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_save_league"/>
    <addaction name="action_load_league"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_save_league">
   <property name="text">
    <string>Save League</string>
   </property>
  </action>
  <action name="action_load_league">
   <property name="text">
    <string>Load League</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
